src/ui/main-window/ThumbnailTitleView.swift

```diff
@@ -60,7 +82,11 @@ class ThumbnailTitleView: BaseLabel {
         return ceil(textSize.width)
     }
 
+    static func extraLineSpacing(for fontSize: CGFloat) -> CGFloat {
+        return fontSize * 0.2
```

Comment:
I imagine that 20% margin comes from your testing? Less than 20% didn't work in your tests?
Thank you